mathjaxandroid/src/main/java/de/timfreiheit/mathjax/android/MathJaxView.java

```diff
@@ -146,8 +160,9 @@ public void setInputText(String inputText) {
         } else {
             laTexString = "";
         }
+        if (onMathJaxRenderListener == null)
```

Comment:
So if the listener is set, then shouldn't we let the user decide on what happens to the webview ? Because if the library itself sets it to invisible, then the use of listener becomes pointless ?